assignment/assignment.hs

```diff
@@ -30,19 +31,22 @@ factorial n = 0 -- replace me
 --
 -- You may also need list concatenation, `++`.
 
-toDigits :: Int -> [Char]
-toDigits num = "replace me"
-
-
+toDigits :: Int -> [Char] -- think of the base case
+toDigits 0 = []        -- is this a sufficent placeholder? Doesn't account for 0 tho          
+toDigits num = toDigits (result) ++ [intToDigit(remainder)] --building from ones to tens to hundreds...
+    where (result, remainder) = divMod num 10           --tried using fst and snd o selected the truncated Result or the remainder, but ran into errors, tried different approach
+                                                              -- and worked
 -------
--- 3 --
+-- 3 --s
 -------
 -- Write a recursive function that counts the number of times
 -- a given value occurs in a list
 
 count :: (Eq a, Num p) => a -> [a] -> p
-count element list = 0 -- replace me
-
+count element [] = 0 -- replace me | no
+count element list = fromEnum (elementn == element) + count element (tail list)  --From Enum should make the boolean an integer, I don't know why it is failing
+  where 
+    elementn = list !! 0
 -------
```

Comment:
That seems to work here.

However, this is another good place for pattern matching, rather than using tail andf `!! 0`.

~~~ hs
count val [] = 0 
count val (x:xs) =
    n + count val xs
    where 
        n = if val == x then 1 else 0
~~~